.gitlab/source_test/linux.yml

```diff
@@ -28,13 +28,6 @@
   script:
     - !reference [.retrieve_linux_go_tools_deps]
     - inv -e install-tools
-    - python3 -m pip install -r tasks/libs/requirements-github.txt
-    - inv -e invoke-unit-tests
-    - pushd test/kitchen
-    - inv -e kitchen.invoke-unit-tests
-    - popd
-    - FAST_TESTS_FLAG=""
-    - if [[ "$FAST_TESTS" == "true" ]]; then FAST_TESTS_FLAG="--only-impacted-packages"; fi
```

Comment:
Is this expected? It sounds like this PR disables the fast tests features introduced in https://github.com/DataDog/datadog-agent/pull/23239.

Reply:
Wow no this is not expected I completely failed my rebase,  many thanks